profiles/databricks.yaml

```diff
@@ -14,5 +14,14 @@
 INGEST_DATA_LOCATION: dbfs:/databricks-datasets/nyctaxi-with-zipcodes/subsampled
 # Specify the format of the dataset
 INGEST_DATA_FORMAT: spark_sql
+# Use a section of the TLC Trip Record Dataset for model development
+# (https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page)
+INGEST_SCORING_DATA_LOCATION: dbfs:/databricks-datasets/nyctaxi-with-zipcodes/subsampled
+# Specify the format of the dataset
+INGEST_SCORING_DATA_FORMAT: spark_sql
 # Override the default train / validation / test dataset split ratios
 SPLIT_RATIOS: [0.75, 0.125, 0.125]
+# Specify the output location of the batch scoring predict step
+SCORED_OUTPUT_DATA_LOCATION: "dbfs:/temp/batch_scoring/sample_output.parquet"
```

Comment:
If I run the pipeline twice, the first output will be overwritten. Is it really desired to define the exact file name?

Reply:
We could write this locally w/in the output dir of the step the way we do for all of the outputs in the training pipeline.  But this does raise a sharp edge here for users that I'm not sure we could get around.

Reply:
We can merge this as is, but keep in mind this will cause problems owing to lack of versioning of output. I'd be much less concerned if it's a Delta table.

Reply:
I think jerry already tested using a Delta table.  Let me check with him on Monday and I'll change this to be a delta table.  I think we will in parallel make sure to write the output as a parquet file w/in the step cache (as the artifact output for the step) though and can test around that.